### PR TITLE
Documentation Update : Lerobot

### PR DESCRIPTION
**Pull request by shantanuparab-tr**
Added documentation for using Aloha Kit with LeRobot. This documentation covers the folllowing:

1. Teleoperation and Setup
2. Recording Episodes
3. Replaying Episodes
4. Visualization of Episodes and Datasets
5. Training and Evaluation
6. Using Pretrained Models for Evaluation
7. Troubleshooting

### CODE REVIEW

File: docs/operation/lerobot_guide.rst [hunk 1]
```diff
@@ -0,0 +1,382 @@
...
+   - Camera Setup: `Aloha Post Install Camera Setup <https://docs.trossenrobotics.com/aloha_docs/getting_started/stationary/software_setup.html#camera-setup>`_
+
+#. Update the serial numbers in the configuration file: :file:`lerobot/common/configs/robot/aloha.yaml`
+
+#. Run the teleoperation script:
+
+   .. code-block:: bash
+
+      $ python lerobot/scripts/control_robot.py teleoperate \
+      $   --robot-path lerobot/configs/robot/aloha.yaml
```
Comment (lukeschmitt-tr) [inline docs/operation/lerobot_guide.rst:89]: Dollar signs only on the line that starts the command

```suggestion
          --robot-path lerobot/configs/robot/aloha.yaml
```

File: docs/operation/lerobot_guide.rst [hunk 2]
```diff
@@ -0,0 +1,382 @@
...
+   $   --root data \
+   $   --repo-id ${HF_USER}/aloha_test \
+   $   --episode 0
+
+.. tip::
+
+   Use different :guilabel:`--fps` values to adjust the frequency of the robot actions.
+
+
+
```
Comment (lukeschmitt-tr) [inline docs/operation/lerobot_guide.rst:224]:
```suggestion
```

File: docs/operation/lerobot_guide.rst [hunk 3]
```diff
@@ -0,0 +1,382 @@
...
+         $ HF_USER=$(huggingface-cli whoami | head -n 1)
+
+Visualizing Datasets
+====================
+
+.. video:: ../videos/visualize_dataset.mp4
+   :width: 1280
+   :height: 720
+
+
```
Comment (lukeschmitt-tr) [inline docs/operation/lerobot_guide.rst:179]:
```suggestion
```

File: docs/operation/lerobot_guide.rst [hunk 4]
```diff
@@ -0,0 +1,382 @@
...
+.. code-block:: bash
+
+   $ DATA_DIR=data python lerobot/scripts/train.py \
+   $   dataset_repo_id=${HF_USER}/aloha_test \
+   $   policy=act_aloha_real \
+   $   env=aloha_real \
+   $   hydra.run.dir=outputs/train/act_aloha_test \
+   $   hydra.job.name=act_aloha_test \
+   $   device=cuda \
+   $   wandb.enable=true
```
Comment (lukeschmitt-tr) [inline docs/operation/lerobot_guide.rst:239]: Maybe keep wandb disabled for the tutorial for simplicity

File: docs/operation/lerobot_guide.rst [hunk 5]
```diff
@@ -0,0 +1,382 @@
...
+   $   --root data \
+   $   --repo-id ${HF_USER}/eval_aloha_test \
+   $   --tags tutorial eval \
+   $   --warmup-time-s 5 \
+   $   --episode-time-s 30 \
+   $   --reset-time-s 30 \
+   $   --num-episodes 10 \
+   $   -p outputs/train/act_aloha_test/checkpoints/last/pretrained_model
+
+
```
Comment (lukeschmitt-tr) [inline docs/operation/lerobot_guide.rst:291]:
```suggestion
```

File: docs/operation/lerobot_guide.rst [hunk 6]
```diff
@@ -0,0 +1,382 @@
...
+This command is similar to the one used for recording training datasets, with a couple of key changes:
+
+#. The :guilabel:`-p` argument is now included, which specifies the path to your policy checkpoint (e.g., :guilabel:`-p outputs/train/eval_aloha_test/checkpoints/last/pretrained_model`).
+   You can also refer to the model repository on Hugging Face if you have uploaded a model checkpoint there (e.g., :guilabel:`-p ${HF_USER}/act_aloha_test`).
+
+#. The dataset name begins with :guilabel:`eval`, reflecting that you are running inference (e.g., :guilabel:`--repo-id ${HF_USER}/eval_aloha_test`).
+
+
```
Comment (lukeschmitt-tr) [inline docs/operation/lerobot_guide.rst:299]:
```suggestion
```

File: docs/operation/lerobot_guide.rst [hunk 7]
```diff
@@ -0,0 +1,382 @@
...
+You can visualize the evaluation dataset afterward using:
+
+.. code-block:: bash
+
+   $ python lerobot/scripts/visualize_dataset.py \
+   $   --root data \
+   $   --repo-id ${HF_USER}/eval_aloha_test
+
+
```
Comment (lukeschmitt-tr) [inline docs/operation/lerobot_guide.rst:308]:
```suggestion
```

File: docs/operation/lerobot_guide.rst [hunk 8]
```diff
@@ -0,0 +1,382 @@
...
+   $  --fps 30 \
+   $  --root data \
+   $  --repo-id ${HF_USER}/eval_aloha_test \
+   $  --tags tutorial eval \
+   $  --warmup-time-s 5 \
+   $  --episode-time-s 30 \
+   $  --reset-time-s 30 \
+   $  --num-episodes 10 \
+   $  -p ${HF_USER}/act_aloha_test
+
```
Comment (lukeschmitt-tr) [inline docs/operation/lerobot_guide.rst:331]:
```suggestion
```

File: docs/operation/lerobot_guide.rst [hunk 9]
```diff
@@ -0,0 +1,382 @@
...
+Datasets can also be downloaded from the Trossen Robotics Community on Hugging Face for further training or data augmentation.
+These datasets can be used with your preferred network architectures.
+Instructions for downloading and using these datasets can be found at the following link:
+
+`Dataset Download and Upload Instructions <https://docs.trossenrobotics.com/aloha_docs/operation/hugging_face.html>`_
+
+`Trossen Robotics Community <https://huggingface.co/TrossenRoboticsCommunity>`_
+
+
```
Comment (lukeschmitt-tr) [inline docs/operation/lerobot_guide.rst:344]:
```suggestion
```

File: docs/operation/lerobot_guide.rst [hunk 10]
```diff
@@ -0,0 +1,382 @@
...
+   Or:
+
+   .. code-block:: bash
+
+      $ brew install ffmpeg
+
+#. **Arrow Keys Not Working During Data Recording (Linux)**
+
+   Ensure that the :guilabel:`$DISPLAY` environment variable is set correctly.
+
```
Comment (lukeschmitt-tr) [inline docs/operation/lerobot_guide.rst:378]:
```suggestion
```

File: docs/operation/lerobot_guide.rst [hunk 11]
```diff
@@ -0,0 +1,382 @@
...
+      $   --reset-time-s 30 \
+      $   --num-episodes 2
+
+.. note::
+
+   #. The :guilabel:`--num-episodes` defines the total number of episodes to be collected.
+      Therefore it will check the existing output directories for any previously recorded episodes and will start recording from the last recorded episode.
+   
+   #. The recorded data is pushed to hugging face hub by default you can set this false by using :guilabel:`--push_to_hub 0`.
+
```
Comment (lukeschmitt-tr) [inline docs/operation/lerobot_guide.rst:156]:
```suggestion
```

**Review by lukeschmitt-tr — CHANGES_REQUESTED** [review]

File: docs/operation/lerobot_guide.rst (hunk 4 above)
Comment (shantanuparab-tr) [reply docs/operation/lerobot_guide.rst]: wandb is disabled as deafult

File: docs/operation/lerobot_guide.rst [hunk 12]
```diff
@@ -0,0 +1,373 @@
...
+      --fps 30 \
+      --root data \
+      --repo-id ${HF_USER}/aloha_test \
+      --episode 0
+
+.. tip::
+
+   Use different :guilabel:`--fps` values to adjust the frequency of the robot actions.
+
+
```
Comment (lukeschmitt-tr) [inline docs/operation/lerobot_guide.rst:222]:
```suggestion
```

**Review by lukeschmitt-tr — CHANGES_REQUESTED** [review]

**Review by lukeschmitt-tr — APPROVED** [review]

**Pull request merged**